lib/signaling/v2/peerconnection.js

```diff
@@ -682,19 +690,34 @@ class PeerConnectionV2 extends StateMachine {
   }
 
   /**
-   * Rewrite local MediaStreamTrack IDs in the given Unified Plan RTCSessionDescription.
+   * Add or rewrite Rewrite local MediaStreamTrack IDs in the given Unified Plan RTCSessionDescription.
```

Comment:
Add or rewrite ~Rewrite~ local MediaStreamTrack IDs in the given Unified Plan RTCSessionDescription.